src/GestureManager.cpp

```diff
@@ -127,6 +128,7 @@ bool GestureManager::handleDragGesture(const DragGestureEvent& gev) {
             if (**WORKSPACE_SWIPE_FINGERS != gev.finger_count) {
                 return false;
             }
+            EMIT_HOOK_EVENT("oppositeGesture", 0);
```

Comment:
```suggestion
        static auto* const PEVENTVEC = g_pHookSystem->getVecForEvent(name);                                                                                                        
        SCallbackInfo      info;                                                                                                                                                   
        g_pHookSystem->emit(PEVENTVEC, info, param);
        if (info.cancelled)
          return true;                                                                                                                           

```

this is a modified version of EMIT_HOOK_EVENT_CANCELLABLE